presto-main/src/test/java/com/facebook/presto/execution/TestSqlTaskManager.java

```diff
@@ -87,43 +90,47 @@ public void testEmptyQuery()
     {
         try (SqlTaskManager sqlTaskManager = createSqlTaskManager(new TaskManagerConfig())) {
             TaskId taskId = TASK_ID;
-            TaskInfo taskInfo = sqlTaskManager.updateTask(TEST_SESSION,
-                    taskId,
-                    Optional.of(PLAN_FRAGMENT),
-                    ImmutableList.of(),
-                    createInitialEmptyOutputBuffers(PARTITIONED)
-                            .withNoMoreBufferIds());
+            TaskInfo taskInfo = createTask(sqlTaskManager, taskId, createInitialEmptyOutputBuffers(PARTITIONED));
             assertEquals(taskInfo.getTaskStatus().getState(), TaskState.RUNNING);
 
             taskInfo = sqlTaskManager.getTaskInfo(taskId);
             assertEquals(taskInfo.getTaskStatus().getState(), TaskState.RUNNING);
 
-            taskInfo = sqlTaskManager.updateTask(TEST_SESSION,
-                    taskId,
-                    Optional.of(PLAN_FRAGMENT),
-                    ImmutableList.of(new TaskSource(TABLE_SCAN_NODE_ID, ImmutableSet.of(), true)),
-                    createInitialEmptyOutputBuffers(PARTITIONED)
-                            .withNoMoreBufferIds());
+            taskInfo = createTask(sqlTaskManager, taskId, ImmutableSet.of(), createInitialEmptyOutputBuffers(PARTITIONED));
             assertEquals(taskInfo.getTaskStatus().getState(), TaskState.FINISHED);
 
             taskInfo = sqlTaskManager.getTaskInfo(taskId);
             assertEquals(taskInfo.getTaskStatus().getState(), TaskState.FINISHED);
         }
     }
 
+    private TaskInfo createTask(SqlTaskManager sqlTaskManager, TaskId taskId, ImmutableSet<ScheduledSplit> splits, OutputBuffers outputBuffers)
```

Comment:
Place this at the bottom of the file near the other creation utility methods like `createSqlTaskManager`